source/_integrations/keenetic_ndms2.markdown

```diff
@@ -2,30 +2,42 @@
 title: Keenetic NDMS2 Routers
 description: Instructions on how to integrate Keenetic NDMS2 Routers into Home Assistant.
 ha_category:
+  - Hub
   - Presence Detection
 ha_release: 0.54
 ha_codeowners:
   - '@foxel'
 ha_domain: keenetic_ndms2
 ---
 
-This integration offers presence detection by examining devices connected to a [Keenetic](https://keenetic.net/)
-Router running NDMS2 firmware.
+This platform offers presence detection by examining devices connected to a [Keenetic](https://keenetic.net/)
+Router running NDMS firmware versions 2.05 and up.
 
-To use a Keenetic router in your installation, add the following to your `configuration.yaml` file:
+<div class='note warning'>
+
+You need to enable telnet on your router for this integration to work.
+
+</div>
```

Comment:
It this really a warning? Or is this just part of pre-requisites in the installation instructions?

How would one enable telnet on the router?

Reply:
Rechecked. It's enabled by default. Since it's router OS version dependant I can't include a hint here. But it's covered in the router manual